### DISCUSSION TITLE
How to achieve different particle spacing

### DISCUSSION THREAD

**Discussion by dreamer0501** [DualSPHysics/DualSPHysics#197] [DUALSPHYSICS_V5.2]
The current version seems to only calculate the uniform particle spacing, but if there are two calculation domains with different particle spacing, can it be achieved?

**Comment by alexbexe** [comment]
Dear user, 
The version 5.2 does not include that option, however we are now working on the new v5.4 that will include variable resolution as you can see in the following examples: https://www.youtube.com/watch?v=46rqz6_kxek&list=PLwaIMU-iIzj0qZVND_JrwAHkg_QG6x-td&pp=gAQBiAQB

More information can be found at: Ricci F, Vacondio R, Tafuni A. 2024. Multiscale Smoothed Particle Hydrodynamics based on a domain-decomposition strategy. Computer Methods in Applied Mechanics and Engineering, 418(A), 116500. [doi:10.1016/j.cma.2023.116500](https://doi.org/10.1016/j.cma.2023.116500).

This version v5.4 is expected to be released next 2025!
